linera-service/src/cli_wrappers/local_kubernetes_net.rs

```diff
@@ -43,10 +43,7 @@ pub struct LocalKubernetesNetConfig {
 
 /// A simplified version of [`LocalKubernetesNetConfig`]
 #[cfg(any(test, feature = "test"))]
-pub struct SharedLocalKubernetesNetTestingConfig {
-    pub network: Network,
-    pub binaries: Option<Option<PathBuf>>,
-}
+pub struct SharedLocalKubernetesNetTestingConfig(LocalKubernetesNetConfig);
```

Comment:
Do we need this? The comment looks outdated.

Reply:
We could make it a `bool` instead. But we need to know whether to use `SHARED_LOCAL_KUBERNETES_TESTING_NET` or not. (I guess we could just _always_ use it, though?)